app/src/routes/constants.js

```diff
@@ -64,6 +66,11 @@ export const pageRendering = {
   [NOT_FOUND]: { component: NotFoundPage, layout: EmptyLayout },
 
   LOGIN_PAGE: { component: LoginPage, layout: EmptyLayout, access: ANONYMOUS_ACCESS },
+  [LOGIN_PAGE_AFTER_REMOVE]: {
+    component: LoginPageAfterRemove,
```

Comment:
Let's name the page as `RemovedAccountPage` or `AccountRemovedPage` as it is not a login page.